src/CMakeLists.txt

```diff
@@ -1045,3 +1045,6 @@ add_subdirectory(script)
 if(WITH_EMBEDDED)
   add_subdirectory(libcephd)
 endif()
+
+configure_file(${CMAKE_SOURCE_DIR}/src/upstart/ceph-osd.conf.in
+  ${CMAKE_SOURCE_DIR}/src/upstart/ceph-osd.conf @ONLY)
```

Comment:
Besides, I don't think I should modify `ceph.spec.in` since `systemd/CMakeLists.txt` already install the systemd configs.